filters/color-overlay/src/ColorOverlayFilter.ts

```diff
@@ -27,15 +27,19 @@ type Color = number | number[] | Float32Array;
 class ColorOverlayFilter extends Filter
 {
     private _color = 0x0;
+    private _alpha = 0;
 
     /**
      * @param {number|Array<number>} [color=0x000000] - The resulting color, as a 3 component RGB e.g. [1.0, 0.5, 1.0]
+     * @param {number} [alpha=0] - The alpha value of the color
      */
-    constructor(color: Color = 0x000000)
+    constructor(color: Color = 0x000000, alpha: number)
```

Comment:
This is marked as optional, but the typing does not indicate that. Would suggest setting the default value here `alpha:number = 1`

Reply:
set it to `1`